tutorials/notebooks/example_keras_export.ipynb

```diff
@@ -0,0 +1,272 @@
+        "                                save_model_path=keras_file_path,\n",
+        "                                quantization_format=mct.exporter.QuantizationFormat.FAKELY_QUANT)"
+      ],
+      "metadata": {
+        "id": "WLyHEEiwGByT"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Notice that the fakely-quantized model has the same size as the quantized exportable model as weights data types are\n",
+        "float.\n",
+        "\n",
+        "\n",
+        "\n",
+        "### TFLite\n",
+        "The tflite serialization format export in two qauntization formats: INT8 and FAKELY_QUANT.\n",
+        "\n",
+        "#### INT8 TFLite\n",
+        "\n",
+        "The model will be exported as a tflite model where weights and activations are represented as 8bit integers."
+      ],
+      "metadata": {
+        "id": "-L1aRxFGGFeF"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import tempfile\n",
+        "\n",
+        "# Path of exported model\n",
+        "_, tflite_file_path = tempfile.mkstemp('.tflite')\n",
+        "\n",
+        "# Use mode KerasExportSerializationFormat.TFLITE for tflite model and quantization_format.INT8.\n",
+        "mct.exporter.keras_export_model(model=quantized_exportable_model,\n",
+        "                                save_model_path=tflite_file_path,\n",
+        "                                serialization_format=mct.exporter.KerasExportSerializationFormat.TFLITE,\n",
+        "                                quantization_format=mct.exporter.QuantizationFormat.INT8)"
+      ],
+      "metadata": {
+        "id": "V4I-p1q5GLzs"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Compare size of float and quantized model:\n"
+      ],
+      "metadata": {
+        "id": "SBqtJV9AGRzN"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "\n",
+        "# Save float model to measure its size\n",
+        "_, float_file_path = tempfile.mkstemp('.keras')\n",
+        "float_model.save(float_file_path)\n",
+        "\n",
+        "print(\"Float model in Mb:\", os.path.getsize(float_file_path) / float(2 ** 20))\n",
+        "print(\"Quantized model in Mb:\", os.path.getsize(tflite_file_path) / float(2 ** 20))\n",
+        "print(f'Compression ratio: {os.path.getsize(float_file_path) / os.path.getsize(tflite_file_path)}')"
+      ],
+      "metadata": {
+        "id": "LInM16OMGUtF"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "\n",
+        "#### Fakely-Quantized TFLite\n",
+        "\n",
+        "The model will be exported as a tflite model where weights and activations are quantized but represented as float.\n",
+        "operators.\n",
```

Comment:
operators? float operators?

Reply:
I'll change it to data types. It will be clearer